VirtualTest/VirtualFunctions/VirtualFunctions.h

```diff
@@ -0,0 +1,278 @@
+#include "Map.h"
```

Comment:
Is this a copy of the other VirtualFunctions? What's the purpose of having two files named the same?

Reply:
Which VirtualFunctions? I though I just there was only one file named "VirtualFunctions.h".

Reply:
VirtualTest/VirtualFunctions/VirtualFunctions.h and Libraries/VirtualFunctions/VirtualFunctions.h